Core/Core.h

```diff
@@ -86,3 +88,19 @@ enum class MemoryExceptionType {
 
 void Core_MemoryException(u32 address, u32 pc, MemoryExceptionType type);
 void Core_Break();
+
+enum class ExceptionType {
+	NONE,
+	MEMORY,
+	BREAK,
+};
```

Comment:
Not a change, but another type may end up being a jump into outer space.  A bit different from a MEMORY exception.

-[Unknown]

Reply:
BAD_EXEC_ADDR maybe?

Reply:
Added it, but not yet used.